_playwright-tests/UI/PopularRepo.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { describe } from 'node:test';
+import { test, type Page } from '@playwright/test';
+import { navigateToRepositories } from './helpers/navHelpers';
+import { deleteAllRepos } from './helpers/deleteRepositories';
+import { closePopupsIfExist } from './helpers/helpers';
+
+describe('Popular Repositories', () => {
+  test('Clean - Delete any current repos that exist', async ({ page }) => {
+    await deleteAllRepos(page);
+  });
+
+  test('Navigate to Popular Repos tab, assert the Add button is displayed', async ({ page }) => {
+    await navigateToRepositories(page);
+    await closePopupsIfExist(page);
+
+    const AddMenuButton = page.locator('[data-ouia-component-id="add_checked_repos-with-snap"]');
+    await page.getByRole('link', { name: 'Popular repositories' }).click();
+
+
+    // Wait for the Add button to become enabled (up to 10 seconds)
+    await AddMenuButton.first().isEnabled({ timeout: 10000 });
+  });
+
+  test('Add popular repos', async ({ page }) => {
+    await navigateToRepositories(page);
+    await closePopupsIfExist(page);
+
+    const AddMenuButton = page.locator('[data-ouia-component-id="add_popular_repo_toggle"]');
+    const AddButton = page.locator('[data-ouia-component-id="add_popular_repo"]');
+    const AddButtonNoSnap = page.locator('[data-ouia-component-id="add-popular_repo_without-snapshotting"]');
+
+    await page.goto('https://stage.foo.redhat.com:1337/insights/content/repositories#SIDs=&tags=');
```

Comment:
Don't navigate like this. We have BASE_URL set and we should respect that. So either just https://playwright.dev/docs/api/class-page#page-goto or Andrew actually created helper function to navigate to repos. (that you already used in this test).

Reply:
oh yes, will fix